bittide/src/Bittide/ClockControl/Si5395J.hs

```diff
@@ -1195,3 +1195,603 @@ testConfig6_200_5_20 = Si539xRegisterMap{..}
     (0x0C, 0x03, 0x00) :>
     (0x0C, 0x07, 0x00) :>
     (0x0C, 0x08, 0x00) :> Nil
+
+
+{-| Configuration for Si5395J with
+  out0a: 200MHz LVDS 1.8V  connected to GTH SMA clk input (clk0 on quad 226)
+  out0:  200MHz LVDS 1.8V  connected to User SMA clk input on node 7 only
+  out9:   20MHZ LVDS 1.8V
+  out9a: 200MHz LVDS 1.8V
+  all of them doing 10ppb steps on Finc/Fdec
+-}
+testConfig6_200_on_0a_and_0 ::  Si5395RegisterMap
+testConfig6_200_on_0a_and_0 = Si539xRegisterMap{..}
+ where
+  configPreamble = (0x0B, 0x24, 0xC0) :> (0x0B, 0x25, 0x00) :> (0x05, 0x40, 0x01) :> Nil
+  configPostamble = (0x05, 0x14, 0x01) :> (0x00, 0x1C, 0x01) :> (0x05, 0x40, 0x00) :> (0x0B, 0x24, 0xC3) :> (0x0B, 0x25, 0x02) :> Nil
+  config =
```

Comment:
There will come a day where we find a better solution than using these static `Vec`s.

But today is not that day